modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/util/StyleBookUtil.java

```diff
@@ -19,28 +23,60 @@
 public class StyleBookUtil {
 
 	public static String getThemeName(
-		CETManager cetManager, long companyId,
-		HttpServletRequest httpServletRequest, String themeId) {
-
-		String name = themeId;
+		long companyId, HttpServletRequest httpServletRequest, String themeId,
+		String themeType) {
 
-		Theme theme = ThemeLocalServiceUtil.fetchTheme(companyId, themeId);
+		if (Objects.equals(themeType, StyleBookConstants.THEME_TYPE_BUNDLE)) {
+			Theme theme = ThemeLocalServiceUtil.fetchTheme(companyId, themeId);
 
-		if (theme != null) {
-			name = LanguageUtil.format(
-				httpServletRequest, "x-theme", theme.getName());
+			if (theme != null) {
+				return LanguageUtil.format(
+					httpServletRequest, "x-theme", theme.getName());
+			}
 		}
-		else {
+		else if (Objects.equals(
+					themeType, StyleBookConstants.THEME_TYPE_THEME_CSS_CET)) {
+
+			CETManager cetManager = _cetManagerSnapshot.get();
+
 			CET cet = cetManager.getCET(companyId, themeId);
 
 			if (cet != null) {
-				name = LanguageUtil.format(
+				return LanguageUtil.format(
 					httpServletRequest, "x-theme-css-client-extension",
 					cet.getName());
 			}
 		}
 
-		return name;
+		return themeId;
```

Comment:
This looks like it changes the logic. If the theme or themeCSS CX has been undeployed we don't want to return just the themeId, we need to return the id subbed in x-theme or x-theme-css-client-extension.

Reply:
Thanks @thiago-buarqque . You're right, it's the same. I misread the old code. Nothing needs to change here. :)